frontend/src/views/host/terminal/terminal/index.vue

```diff
@@ -436,6 +436,12 @@ onMounted(() => {
         color: var(--panel-terminal-tag-active-text-color);
         background-color: var(--panel-terminal-tag-active-bg-color);
     }
+    :deep(.el-tabs__item:hover) {
+        color: var(--panel-terminal-tag-hover-text-color);
+    }
+    :deep(.el-tabs__item.is-active:hover) {
+        color: var(--panel-terminal-tag-active-text-color);
+    }
 }
 
 .tagButton {
```

Comment:
This code appears to be related to styling components used within Electron apps developed with Electron.js and React. The key differences from 2021-09-01 might include:

- There is no reference to the date `2024-11-27`
- No specific issue, potential issue or optimization suggestion has been provided

If you have specific questions about how various parts of this code work together, please let me know!